app/src/components/keyCloakServiceClientMgr.js

```diff
@@ -168,6 +168,26 @@ class KeyCloakServiceClientManager {
       return undefined;
     }
   }
+
+  async fetchAllClients(kcEnv) {
+
+    log.info('KeyCloakServiceClientManager.fetchAllClients in ' + kcEnv);
+
+    //get all service clients
+    let clients = await this.svc.getClients();
+
+    // remove those that dont match '*_SERVICE_CLIENT';
+    const regex = '.*_SERVICE_CLIENT$';
+    clients = clients.filter(cl => cl.clientId.match(regex));
+
+    // add the keycloak environment to each service client object in the array
+    clients.forEach(function (cl) {
```

Comment:
See comment below about not needing the env passed down.

Reply:
Also this field is called 'environment' in the spec for this new endpoint, not 'realm'